src/operation/emulation_model.jl

```diff
@@ -235,28 +237,18 @@ function build_pre_step!(model::EmulationModel)
             @info "EmulationProblem status not BuildStatus.EMPTY. Resetting"
             reset!(model)
         end
-        # Initial time are set here because the information is specified in the
```

Comment:
I moved this code to the section where we perform other container initialization.